src/SSHLibrary/abstractclient.py

```diff
@@ -159,7 +159,7 @@ def close(self):
         except AttributeError:
             pass
 
-    def login(self, username, password, allow_agent=False, look_for_keys=False, delay=None):
+    def login(self, username, password, allow_agent=False, look_for_keys=False, delay=None,proxy_cmd=None):
```

Comment:
SSHLibrary uses PEP8 style for Python code, and all arguments must be separated with comma and a **space character**